config/module.config.php

```diff
@@ -2,11 +2,11 @@
 return [
     'service_manager' => [
         'invokables' => [
-            'Dash\Router\Http\Parser\ParserManager' => 'Dash\Router\Http\Parser\ParserManager',
-            'Dash\Router\Http\Route\RouteManager' => 'Dash\Router\Http\Route\RouteManager',
+            Dash\Router\Parser\ParserManager::class => Dash\Router\Parser\ParserManager::class,
```

Comment:
I don't see a huge benefit here in the config. Eventually such a config would end up with hundreds of use statements, which just sounds wrong.